src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+:root {
+  --second: 1s;
+  --minute: calc(var(--second) * 60);
+  --hour: calc(var(--minute) * 60);
+}
```

Comment:
sass variable are simpler if you are not going to change them using javascript

Reply:
Here I'll definitely need some explaining

Reply:
calc works in browser, but sass is calculated beforehand
so if you don't need this calculation to relate on some `vm` or `%` please use SCSS variables

Reply:
Thank you for explaining